templates/azure/terraform/expand_property_method.erb

```diff
@@ -74,6 +74,7 @@ func expand<%= sdk_marshal.resource -%><%= descriptor.func_name -%>(input <%= go
 <%        elsif property.is_a?(Api::Type::Array) -%>
     results := make([]<%= sdk_marshal.package -%>.<%= sdk_marshal.sdktype.go_type_name -%>, 0)
     for _, v := range input {
```

Comment:
```suggestion
     for _, item := range input {
```
And rename the original `item` to `result`.

Reply:
ok.